addons/hr/wizard/mail_activity_schedule.py

```diff
@@ -39,13 +37,12 @@ def _compute_plan_summary(self):
         )
         for scheduler in self:
             templates_by_responsible_type = scheduler.plan_id.template_ids.grouped('responsible_type')
-            scheduler.plan_summary = Markup('<ul>%(summary_by_responsible)s</ul>') % {
-                'summary_by_responsible': Markup().join(
-                    Markup("%(responsible)s %(summary_lines)s") % {
-                        'responsible': responsible_value_to_label[key],
-                        'summary_lines': scheduler._get_summary_lines(templates)
-                    } for key, templates in templates_by_responsible_type.items()
-                )}
+            scheduler.plan_summary = self.env['mail.activity.schedule.summary']
+            for key, templates in templates_by_responsible_type.items():
```

Comment:
Do we still need an hr override ? Cant' we better integrate responsible computation in base code ? Seems anyway we want the same display in all apps, considering the specs ?